### PR TITLE
Closes #241 : Show the "Products to be completed" page 

### LINKED ISSUE #241: Show the "Products to be completed" page in a Chrome Custom Tab
http://world.openfoodfacts.org/state/to-be-completed

- This shows the latest products added on Open Food Facts that need to be completed. 
- Useful for demos, for admins (monitor issues)
- Useful to get instant feedback/gratification for your contribution.

### PR DESCRIPTION
**Pull request by PrajwalM2212**
## Description

Displays products to be completed in a recycler view

## Related issues and discussion
#241 
 
 ## Screen-shots, if any
 
 
image
screenshot_1523417028

### CODE REVIEW

**Comment by teolemon** [comment]
Can you rebase on latest trunk ? It should fix the build.

**Comment by PrajwalM2212** [comment]
I have done the rebase

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -555,5 +555,6 @@
     <string name="notify_content">Click to view exported products</string>
     <string name="notification_channel_name">Export Notifications</string>
     <string name="notify_channel_description">Notification to show exported file\'s location</string>
+    <string name="products_to_be_completed">Products to be Completed</string>
```
Comment (Karljoones) [inline app/src/main/res/values/strings.xml]: This is tiny but can you remove the capital C from Completed? Just to be more consistent with the rest of the UI. Once that is done, ready for merging.

**Comment by PrajwalM2212** [comment]
@Karljoones I have made the change and resolved the conflicts

**Comment by teolemon** [comment]
@PrajwalM2212 Can you rebase ? 
@ross-holloway94 @Karljoones Looks good to me ux-wise.

**Comment by PrajwalM2212** [comment]
I will rebase by today night .

**Comment by PrajwalM2212** [comment]
@ross-holloway94 @teolemon  Rebase done

**Comment by Karljoones** [comment]
Merged, thanks @PrajwalM2212

**Pull request merged**